packages/teams-js/test/public/location.spec.ts

```diff
@@ -28,203 +29,301 @@ describe('location', () => {
-    expect(message).not.toBeNull();
-    expect(message.args.length).toBe(1);
-    expect(message.args[0]).toEqual(defaultLocation);
-  });
-  it('showLocation call in content frameContext works', async () => {
-    await framelessPlatform.initializeWithContext(FrameContexts.content);
-    framelessPlatform.setClientSupportedSDKVersion(minVersionForLocationAPIs);
-    framedPlatform.setRuntimeConfig({ apiVersion: 1, supports: { location: {} } });
-    location.showLocation(defaultLocation);
-    const message = framelessPlatform.findMessageByFunc('location.showLocation');
-    expect(message).not.toBeNull();
-    expect(message.args.length).toBe(1);
-    expect(message.args[0]).toEqual(defaultLocation);
-  });
-  it('showLocation calls with successful result', async () => {
-    await framelessPlatform.initializeWithContext(FrameContexts.content);
-    framelessPlatform.setClientSupportedSDKVersion(minVersionForLocationAPIs);
-    framedPlatform.setRuntimeConfig({ apiVersion: 1, supports: { location: {} } });
-    const promise = location.showLocation(defaultLocation);
-
-    const message = framelessPlatform.findMessageByFunc('location.showLocation');
-    expect(message).not.toBeNull();
-    expect(message.args.length).toBe(1);
-    expect(message.args[0]).toEqual(defaultLocation);
-
-    const callbackId = message.id;
-    framelessPlatform.respondToMessage({
-      data: {
-        id: callbackId,
-        args: [undefined, true],
-      },
-    } as DOMMessageEvent);
-
-    return expect(promise).resolves;
-  });
-  it('showLocation calls with error', async () => {
-    await framelessPlatform.initializeWithContext(FrameContexts.content);
-    framelessPlatform.setClientSupportedSDKVersion(minVersionForLocationAPIs);
-    framedPlatform.setRuntimeConfig({ apiVersion: 1, supports: { location: {} } });
-    const promise = location.showLocation(defaultLocation);
-
-    const message = framelessPlatform.findMessageByFunc('location.showLocation');
-    expect(message).not.toBeNull();
-    expect(message.args.length).toBe(1);
-    expect(message.args[0]).toEqual(defaultLocation);
-
-    const callbackId = message.id;
-    framelessPlatform.respondToMessage({
-      data: {
-        id: callbackId,
-        args: [{ errorCode: ErrorCode.PERMISSION_DENIED }],
-      },
-    } as DOMMessageEvent);
-
-    await expect(promise).rejects.toEqual({ errorCode: ErrorCode.PERMISSION_DENIED });
-  });
-  it('Frameless - getLocation should throw error when not supported in the runtime config', async () => {
-    await framelessPlatform.initializeWithContext(FrameContexts.task);
-    framedPlatform.setRuntimeConfig({ apiVersion: 1, supports: {} });
-    const promise = location.getLocation(defaultLocationProps);
-    await expect(promise).rejects.toEqual(errorNotSupportedOnPlatform);
-  });
-  it('Frameless - showLocation should throw error when location is not supported', async () => {
```

Comment:
Even though it's deprecated, we should still have tests for `location.isSupported`. I would have expected this file to look a little more like locationLegacy.spec.ts, which does have those tests I think. (I diff'ed location.spec.ts with your changes against locationLegacy.spec.ts which was helpful in reviewing :)).